src/pods/test-verb-forms/components/verb-audio.component.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { Select, Button, FormControl, MenuItem } from '@material-ui/core';
+import { Verb } from '../test-verb-forms.vm';
+import { useSpeechSynthesis } from 'react-speech-kit';
+
+interface Props {
+  verb: Verb;
+}
+
+const VOICE_RATE = 0.8;
+const VOICE_PITCH = 1;
+
+export const VerbAudio: React.FC<Props> = props => {
+  const { verb } = props;
+  const { speak, voices, supported, speaking } = useSpeechSynthesis();
+  const [voiceIndex, setVoiceIndex] = React.useState('0');
+  const [englishVoicesArray, setEnglishVoicesArray] = React.useState([]);
+  let voice = englishVoicesArray[voiceIndex] || null;
+
+  React.useEffect(() => {
+    setEnglishVoicesArray(
+      voices.filter(voice => {
+        return voice.lang.substring(0, 2) === 'en';
+      })
```

Comment:
We have to work in a defensive way here, many scenarios can arose and application could become buggy or crash.

Let's do the following thing:
- What would happen if there's no 'en' voice ? It should be empty, then we should disable or hide the play button.

Reply:
Here instead of choosing 0, for english voices I would search for the first voice that is en-gb, in case it doesn't exists fallback to english-us if not then take the first one that is available

Reply:
I would extract to a business method and unit test it using jest